### ISSUE TITLE
Found with fuzzer: gaze attack 65?

### ISSUE THREAD

**Issue by elunna** [elunna/hackem#16]
```

Program received signal SIGABRT, Aborted.
__GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
50	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
#1  0x00007ffff7365859 in __GI_abort () at abort.c:79
#2  0x0000555555a428ca in NH_abort () at end.c:236
#3  0x0000555555a496c6 in panic (str=0x55555651cb00 "%s") at end.c:802
#4  0x0000555555f9765b in impossible (s=0x55555651f680 "gaze attack %d?") at pline.c:518
#5  0x0000555555fa9722 in dogaze () at polyself.c:1575
#6  0x000055555582dbf6 in domonability () at cmd.c:726
#7  0x0000555555860cd7 in rhack (cmd=0x55555679cce0 <in_line> <incomplete sequence \355>)
    at cmd.c:5517
#8  0x0000555555762c00 in moveloop (resuming=0 '\000') at allmain.c:798
#9  0x00005555563b5144 in main (argc=0, argv=0x7fffffffe658) at ../sys/unix/unixmain.c:353
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
        set = {__val = {0, 0, 0, 0, 0, 140737337632192, 93824997402012, 335544320, 
            140737341055120, 1073741824, 0, 140737342801568, 4294967295, 140737337360384, 
            140737342784672, 13590347967293169408}}
        pid = <optimized out>
        tid = <optimized out>
        ret = <optimized out>
#1  0x00007ffff7365859 in __GI_abort () at abort.c:79
        save_stage = 1
        act = {__sigaction_handler = {sa_handler = 0xffffffff, sa_sigaction = 0xffffffff}, 
          sa_mask = {__val = {140737337360384, 140737342784672, 13590347967293169408, 
              140737488344848, 140737342801568, 18446744073709547520, 140737327112192, 
              140737345470680, 140737488345920, 140737488345920, 140737344339986, 
              17592186043106, 140737488338944, 140737341055120, 13590347967293169408, 
              140737488345248}}, sa_flags = -1267327232, sa_restorer = 0x7fffffffda60}
        sigs = {__val = {32, 0 <repeats 15 times>}}
#2  0x0000555555a428ca in NH_abort () at end.c:236
        gdb_prio = 1
        libc_prio = 2
        aborting = 1 '\001'
#3  0x0000555555a496c6 in panic (str=0x55555651cb00 "%s") at end.c:802
        the_args = {{gp_offset = 16, fp_offset = 48, overflow_arg_area = 0x7fffffffdb30, 
            reg_save_area = 0x7fffffffda60}}
#4  0x0000555555f9765b in impossible (s=0x55555651f680 "gaze attack %d?") at pline.c:518
        pbuf = "gaze attack 65?\000`\342\377\377\377\177\000\000\320\342\377\377\377\177\000\000\060\343\377\377\377\177\000\000Z\374\377\377\377\017\000\000\320\342\377\377\377\177\000\000\217\341T\367\377\177", '\000' <repeats 1217 times>
        the_args = {{gp_offset = 16, fp_offset = 48, overflow_arg_area = 0x7fffffffe220, 
            reg_save_area = 0x7fffffffe140}}
#5  0x0000555555fa9722 in dogaze () at polyself.c:1575
        mtmp = 0x55555679cce0 <in_line>
        looked = 0
        qbuf = "\200\342\377\377\377\177\000\000\377\321;VUU\000\000\200\342\377\377\377\177\000\000\020\001\000\000\320`\000\000\300\342\377\377\377\177\000\000$\346<VUU\000\000P\344\377\377\377\177\000\000\000\037v\264\001", '\000' <repeats 11 times>, "\320\342\377\377\377\177\000\000\060\343\377\377\377\177\000\000\355\377\377\377\000\000\000\000`\343\377\377\377\177\000\000и\206UUU\000\000\263\212\265A\000\000\000\000@,EVUU\000"
        i = 0
        adtyp = 65 'A'
#6  0x000055555582dbf6 in domonability () at cmd.c:726
No locals.
#7  0x0000555555860cd7 in rhack (cmd=0x55555679cce0 <in_line> <incomplete sequence \355>)
    at cmd.c:5517
        tlist = 0x55555665d0e0 <extcmdlist+1824>
        res = 1102416563
        func = 0x55555582da6d <domonability>
        spkey = 0
        prefix_seen = 0 '\000'
        bad_command = 85 'U'
        firsttime = 1 '\001'
#8  0x0000555555762c00 in moveloop (resuming=0 '\000') at allmain.c:798
        moveamt = 21
        wtcap = 0
        change = 0
        monscanmove = 0 '\000'
        timeout_start = 26543
        past_clock = -11869
        elf_regen = 1 '\001'
        orc_regen = 1 '\001'
        vamp_regen = 1 '\001'
#9  0x00005555563b5144 in main (argc=0, argv=0x7fffffffe658) at ../sys/unix/unixmain.c:353
        fd = -1
        dir = 0x0
        exact_username = 0 '\000'
        resuming = 0 '\000'
        plsel_once = 1 '\001'
```

**Comment by elunna** [comment]
Happening on the player side in polyself.c

#define AD_FEAR 65 /* inflict fear (eye of fear and flame) */

Not implemented as a player #monster ability.

**Comment by elunna** [comment]
Fixed in this commit: 13241d52d
